.readthedocs.yml

```diff
@@ -3,7 +3,13 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
 # Required
-version: 1.4
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-lts-latest
+  tools:
+    python: "3.12"
```

Comment:
Why don't we use "latest" or "3" (latest python3) ?

Reply:
I just copied from the https://docs.readthedocs.io/en/stable/config-file/v2.html example.
Changed to "3" now.